Логинов Андрей

AndrewLoginov opened the pull request:
Решение(1)
[Посмотреть решение](https://hruview.herokuapp.com/AndrewLoginov/markup-task-5/master/index.html)

AndrewLoginov committed:
Логинов Андрей

Решение(1)

AndrewLoginov committed:
Логинов Андрей

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

mokhov commented on index.css:
```diff
@@ -0,0 +1,104 @@
+    border-bottom: 1px solid #8a8f99;
+    margin-bottom: 10px;
+    width: 100%;
+    position: fixed;
```
Хорошо бы не использовать position: fixed

mokhov reviewed (changes requested):
1. В такой вёрстке (когда всегда есть хедер и футер) не принято чтобы был вертикальный скролл, когда всё влазит по высоте
2. Крестик, закрывающий попап при ховере становится не заметным (черным). Выглядит как баг

В остальном очень неплохо, но пока 🍅

AndrewLoginov committed:
Решение(2)

1) Убрал вертикальный скролл
2) Убрал изменение цвета крестика при ховере.
3) Закрепил header  с помощью position: sticky

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
